src/react/tests/useQuery.test.tsx

```diff
@@ -4677,4 +4677,38 @@ describe('useQuery', () => {
 
     consoleMock.mockRestore()
   })
+
+  it('errorUpdateCount should increased on each refetch', async () => {
+    const consoleMock = mockConsoleError()
+    const key = queryKey()
+    const error = new Error('oops')
+
+    function Page() {
+      const { refetch, errorUpdateCount } = useQuery(
+        key,
+        async () => {
+          throw error;
+        },
+        {
+          retry: false,
+        }
+      )
+      return <div>
+        <button onClick={() => refetch()}>refetch</button>
+        <span>
+          data: {errorUpdateCount}
+        </span>
+        </div>
+    }
+    const rendered = renderWithClient(queryClient, <Page />)
+    const fetchBtn = rendered.getByRole('button', { name: 'refetch' })
+    fireEvent.click(fetchBtn)
+    await waitFor(() => rendered.getByText('data: 1'))
+    fireEvent.click(fetchBtn)
+    await waitFor(() => rendered.getByText('data: 2'))
+    fireEvent.click(fetchBtn)
+    await waitFor(() => rendered.getByText('data: 3'))
```

Comment:
since the query is enabled per default, I think we should wait for the count to become `1` before we click refetch the first time. the error count is not about refetches - its about all errors :)

Reply:
Thanks, will update the test :)